quizes.py

```diff
@@ -74,7 +75,8 @@ class MADRSQuiz(BaseQuiz):
 
     RESULTS = {
         'en': ['normal 👍', 'mild depression 😐', 'moderate depression 😔', 'severe depression 😨'],
-        'ru': ['норма 👍', 'слабая депрессия 😐', 'умеренная депрессия 😔', 'тяжелая депрессия 😨']}
+        'ru': ['норма 👍', 'слабая депрессия 😐', 'умеренная депрессия 😔', 'тяжелая депрессия 😨'],
+        'en': ['normal 👍', 'depressão leve 😐', 'depressão moderada 😔', 'depressão severa 😨']}
```

Comment:
Wrong key, there should be `pt` instead of `en`

Reply:
Fixed language key previous conflict.